### ISSUE TITLE
Text animation is misleading when re-opening fragment

### ISSUE THREAD

**Issue by agilob** [CellularPrivacy/Android-IMSI-Catcher-Detector#849]
If animation was supposed to change color of values that changed, then it should not animate all labels on step 3:

1. Phone/SIM Details
2. Current Threat Level
3. Phone/SIM Details (nothing changed but all values are animated - misleading and hides values that changed in meantime)

**Comment by SecUpwN** [comment]
@kaichunlin, since you solved #288, could you please improve this to only highlight real changes?

**Comment by kaichunlin** [comment]
@SecUpwN yeah I'll check into it

**Comment by SecUpwN** [comment]
Thanks @kaichunlin, all pull requests from you to fix this and other UI/UX Issues are greatly appreciated!

**Comment by SecUpwN** [comment]
@kaichunlin, are you still on it?

**Comment by kaichunlin** [comment]
@SecUpwN yeah... just got a pretty big work/life plate to finish first. I will work on it next week, promise :+1:

**Comment by SecUpwN** [comment]
>yeah... just got a pretty big work/life plate to finish first. I will work on it next week, promise

No need to promise anything, @kaichunlin. I just wanted to make sure you have it on your radar, don't feel pushed. I am thankful for any contribution! And just as a hint: We have [Issues with Bounties](https://github.com/CellularPrivacy/Android-IMSI-Catcher-Detector/issues?q=is%3Aopen+is%3Aissue+label%3Abounty) as well.

**Comment by SecUpwN** [comment]
@kaichunlin, please make sure to **delete** your old fork an re-fork our project before adding your PR!

**Comment by kaichunlin** [comment]
@SecUpwN I can't seem to find the option "Current Threat Level" in the app built from the development branch. Is it part of the hamburger menu options?

**Comment by SecUpwN** [comment]
@kaichunlin, thanks for having a look! I've changed it to fix #848. Do you need any other information?

**Comment by kaichunlin** [comment]
@SecUpwN Since the fragment is created anew when switching between menu options, the last displayed info has to be stored elsewhere (in memory) so it can be compared to when the user switched back to DeviceFragment. Does the app currently have a place to store data that persists between Fragments/Activities?
